handlers/mentionHandler.js

```diff
@@ -0,0 +1,30 @@
+const Discord = require('discord.js');
+const MentionsList = {
```

Comment:
`MentionsList` -> `mentionsList`, albo `MENTIONS_LIST` (w sumie zamiast "list" bardziej pasuje "dict", bo to nie jest lista - ale to szczegół). Jeśli ten słownik jest traktowany jako stała, to można go też wrzucić w [`Object.freeze`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze).

```js
const MENTIONS_DICT = Object.freeze({
  EVERYONE: '@everyone',
  HERE: '@here',
});
```